pkg/reconciler/ksvc/reconciler.go

```diff
@@ -48,9 +48,9 @@ func (c *ServiceReconciler) Reconcile(ctx context.Context, desiredService *knser
 	err := c.client.Get(context.TODO(), types.NamespacedName{Name: desiredService.Name, Namespace: desiredService.Namespace}, service)
 	if err != nil {
 		if errors.IsNotFound(err) {
-		  log.Info("Creating Knative Serving Service", "namespace", service.Namespace, "name", service.Name)
```

Comment:
@yuzisun needs an autoformatter ;)

Reply:
Should the log be "Creating KF Serving Service Instance" ?

Reply:
oops, might forgot to run `make fmt`, need to force this in CI.